npm/builder.js

```diff
@@ -139,13 +139,7 @@ Builder.prototype.release = function (releaseVersion) {
 
   async.series([
     function (callback) { builder.prepareRelease(releaseVersion, callback); },
-    function (callback) { builder.build(callback); },
```

Comment:
I've removed the build and test tasks but I could use, npm `preversion`, `version` and `postversion` scripts ?

```js
"scripts": {
  "preversion": "npm test",
  "version": "npm run build",
  "postversion": "git push && git push --tags"
}
```